incubator/tensorflow-serving/Chart.yaml

```diff
@@ -0,0 +1,10 @@
+name: tensorflow-serving
+home: https://github.com/kubernetes/charts
+version: 0.1.0
+description: TensorFlow Serving is an open-source software library for serving machine learning models.
+sources:
+- https://github.com/tensorflow/serving
+- https://tensorflow.github.io/serving/serving_inception.html
+maintainers:
+- name: Yang Che
+  email: cheyang@163.com
```

Comment:
Build requirements have become a bit stricter, you now need to add an appVersion as well.